docs/synapse/userguides/storm_adv_control.rstorm

```diff
@@ -462,13 +706,27 @@ In this variation on our original query, we isolate the for loop within a subque
       
   $lib.print('And we're done!')
 
-The query performs the same actions as described above, but thanks to the subquery, the behavior of this query is different, as we can see from the query's output:
+The query performs the same actions as described above, but thanks to the subquery, the behavior of this query
+is different, as we can see from the query's output:
 
 .. storm-cli:: MULTILINE=ENRICH
 
-In this case, the query behaves more "as expected" - the strings within the for loop print once for each item / iteration of the loop, ``And we're done!`` prints once, and a single FQDN node exits our pipeline when our query completes. So what's different?
+In this case, the query behaves more "as expected" - the strings within the for loop print once for each item /
+iteration of the loop, ``And we're done!`` prints once, and a single FQDN node exits our pipeline when our query
+completes. So what's different?
 
-One of the key features of a subquery is that by default (i.e., unless the ``yield`` option is used), **the nodes that go into a subquery also come out of a subquery,** regardless of what occurs inside the subquery itself. In other words, **subqueries do not "consume" nodes**. 
-
-We still have our single FQDN inbound to the subquery. Inside the subquery, our for loop still executes, effectively "splitting" the Storm pipeline into three pipelines that execute in sequence. But once we complete the for loop and exit the subquery, those pipelines are "discarded". The single FQDN that went into the subquery exits the subquery. We are back to our single node in the main pipeline. That single node causes our print statement to print ``And we're done!`` only once, and we are left with our single node at the end of the query.
+One of the key features of a subquery is that by default (i.e., unless the ``yield`` option is used), **the nodes
+that go into a subquery also come out of a subquery,** regardless of what occurs inside the subquery itself. In
+other words, **subqueries do not "consume" nodes**, and any changes that occur within the subquery do not exit
```

Comment:
may be good to clarify that this means "changes to the set of nodes in pipeline" or something, because if you change a node (add a prop/tag) you do end up with a modified node. maybe that's obvious though?

Reply:
Went back to original / current (simpler) explanation.